MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,291 @@
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        stepProgressItem = new ObservableCollection<StepProgressBarItem>();
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Completed", SecondaryText = "Task" });
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to step progressbar
+
+The Step progressbar control allows you to bind item collection by setting the [ItemsSource]() property from the [SfStepProgressBar](). Bind item collection in both XAML and C#.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="6" %}
+
+<stepProgressBar:SfStepProgressBar
+                    x:Name="stepProgressBar"
+                    Margin="24,10,0,0"
+                    LabelSpacing="12"
+                    Orientation="Horizontal"  
+                    ItemsSource="{Binding StepProgressItem}">
+</stepProgressBar:SfStepProgressBar>
+
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="7" %}
+
+ViewModel viewModel = new ViewModel();
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    Margin = new Thickness(24, 10, 0, 0),
+    LabelSpacing = 12,
+    Orientation = StepProgressBarOrientation.Horizontal,
+    ItemsSource = viewModel.StepProgressItem,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Formatted text
+To customize the description with different formatting style, [PrimaryFormattedText]() and [SecondaryFormattedText]() can be used. The following code example explains how to set `PrimaryFormattedText` and `SecondaryFormattedText` to a step view.
+
+* Create a simple Observable Collection of item for the step progressbar,
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="34" %}
+
+public class ViewModel
+{
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    private ObservableCollection<StepProgressBarItem> stepProgressItem;
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        FormattedString primaryFormattedText = new FormattedString();
+        primaryFormattedText.Spans.Add(new Span { Text = "Ordered and Approved", FontSize = 13 });
+        primaryFormattedText.Spans.Add(new Span { Text = "&#Your Order has been placed", FontSize = 12 });
```

Comment:
Cheked this code in simplen sample?

Reply:
I have checked the above code in simple sample